awx/ui_next/src/screens/Schedule/ScheduleList/ScheduleList.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { SchedulesAPI } from '@api';
+import { Card, PageSection } from '@patternfly/react-core';
+import AlertModal from '@components/AlertModal';
```

Comment:
I think @marshmalien  is working on removing this component in favor of PF's alert modal